src/components/cards/cardEmptyState/cardEmptyState.tsx

```diff
@@ -0,0 +1,12 @@
+import { EmptyState, type IEmptyStateProps } from '../../states/emptyState';
+import { Card, type ICardProps } from '../card';
+
+export type CardEmptyStateProps = ICardProps & IEmptyStateProps;
+
+export const CardEmptyState: React.FC<CardEmptyStateProps> = (props) => {
```

Comment:
Props interfaces/types always starts with `I`
```suggestion
export const CardEmptyState: React.FC<ICardEmptyStateProps> = (props) => {
```